src/style.css

```
@@ -0,0 +1,29 @@
+*,
+:after,
+:before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.starline {
+  display: flex;
+}
+
+.stars__star-active {
+  background-image: url(/images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-position: 50%;
+  background-repeat: no-repeat;
+}
```

The block with stars--N modifier should have exactly N first stars active, according to [requirements](https://github.com/mate-academy/layout_stars#the-task)

Please try to make it work with `nth-child` pseudoclass

```suggestion
.stars--1 .stars__star:nth-child(-n+1), {
  ... more selectors to target every line of stars ... 
  background-image: url("./images/star-active.svg");
}
```